internal/pkg/docker/docker.go

```diff
@@ -21,6 +26,23 @@ type runner interface {
 	Run(name string, args []string, options ...command.Option) error
 }
 
+// FilePath is the path of a Dockerfile.
+type FilePath string
+
+// parse returns the Dockerfile name and the path where Dockerfile resides.
+// Example: ./feDockerfile returns feDockerfile and .
+func (f FilePath) parse() (string, string) {
```

Comment:
Have you considered using [this](https://golang.org/pkg/path/filepath/#Split)?

Reply:
This is awesome!